AwsCryptographicMaterialProviders/runtimes/net/MPL.csproj

```diff
@@ -56,7 +56,7 @@
   </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="DafnyRuntime" Version="$(DafnyVersion)" />
+        <PackageReference Include="DafnyRuntime" Version="[4.2.0,$(DafnyVersion)]" />
```

Comment:
Is this saying 4.2 is the minimum allowed version? That can't be true anymore, right?

Reply:
This has to be true; otherwise it does not work with the current dbesdk and esdk. As soon as the esdk and dbesdk are bumped to 4.8.0 then we can drop this 4.2.0 inclusive.